doc/Extensions/Applications.md

```diff
@@ -310,6 +310,30 @@ chmod +x /usr/lib/check_mk_agent/local/bind
 
 3. Set the variable 'agent' to '1' in the config.
 
+## BIRD2
+
+The BIRD Internet Routing Daemon (BGP) 
+
+Due to the lack of SNMP support in the BIRD daemon, this application extracts all configured BGP protocols and parses it into LibreNMS.
+This application supports both IPv4 and IPv6 Peer processing.
+
+### SNMP Extend
+1. Edit your snmpd.conf file (usually /etc/snmp/snmpd.conf) and add:
+```
+extend bird2 '/usr/bin/sudo /usr/sbin/birdc -r show protocols all'
+```
+
+2.  Edit your sudo users (usually `visudo`) and add at the bottom:
+
+```
+Debian-snmp ALL=(ALL) NOPASSWD: /usr/sbin/birdc
```

Comment:
Might want to be a little more restrictive and specify arguments.

Reply:
@murrant birdc (interactive shell to print out status) needs to be used here, not sure how to further add safety restrictions, what do you think?